python/cuda_parallel/tests/conftest.py

```diff
@@ -67,3 +67,20 @@ def ptr(self):
 @pytest.fixture(scope="function")
 def cuda_stream() -> Stream:
     return Stream(cp.cuda.Stream())
+
+
+@pytest.fixture(scope="session")
+def monkeysession():
+    with pytest.MonkeyPatch.context() as mp:
+        yield mp
+
+
+@pytest.fixture(scope="session", autouse=True)
+def verify_sass(monkeysession):
+    import cuda.parallel.experimental._cccl_interop
+
+    monkeysession.setattr(
+        cuda.parallel.experimental._cccl_interop,
+        "_check_sass",
+        False,  # todo: change to True
```

Comment:
Note to reviewers: changing this to `True` is what ensures that compile results are checked for LDL/STL instructions.